src/desktop/docks/brushpalettedock.cpp

```diff
@@ -119,7 +143,7 @@ BrushPalette::BrushPalette(QWidget *parent)
 		});
 	}
 
-	d->presetListView = new QListView(this);
+	d->presetListView = new NonScrollingListView(this);
```

Comment:
`IgnoreEnsureVisibleListView`

Reply:
Applied and added a comment with why that even exists, since it's not obvious.